app/assets/stylesheets/govuk_publishing_components/component_guide.scss

```diff
@@ -132,3 +136,34 @@ $border-color: #ccc;
     }
   }
 }
+
+// Preview Page Styling
+html {
+  background: $white;
```

Comment:
Prevents something like this from happening:

<img width="1412" alt="screen shot 2017-08-14 at 11 29 25" src="https://user-images.githubusercontent.com/29889908/29268236-d9c09eec-80e3-11e7-92cf-69ac0667361a.png">